robottelo/config/__init__.py

```diff
@@ -40,10 +40,12 @@ def get_settings():
         try:
             settings.validators.validate()
         except ValidationError as err:
-            logger.warning(
-                f'Dynaconf validation failed, continuing for the sake of unit tests\n{err}'
-            )
-
+            if settings.robottelo.settings.get('skip_validation'):
```

Comment:
Is this really skipping the validation? Wouldn't it be more of a `IGNORE_VALIDATION_ERRORS`?

Reply:
You are right. I was too much focused on keeping the original message. 

What is better?
1. Run validation every time and show the validation error(s) or
2. run validation only if settings tell us to validate?

Reply:
Personally I would prefer 1.

Reply:
Addressed. PTAL